app/graphql/types/subscriptions/charge_overrides_input.rb

```diff
@@ -4,6 +4,7 @@ module Types
   module Subscriptions
     class ChargeOverridesInput < Types::BaseInputObject
       argument :id, ID, required: true
+      argument :billable_metric_id, ID, required: true
```

Comment:
GraphQL/OrderedArguments: Arguments should be sorted in an alphabetical order within their section. Field `billable_metric_id` should appear before `id`.